services/terrain/src/terrain/services/admin.clj

```diff
@@ -40,11 +40,14 @@
   [url-to-scrub]
   (str (url/url url-to-scrub :path "/")))
 
+(defn get-with-timeout
+  [url]
+  (client/get url {:socket-timeout 1000 :conn-timeout 1000}))
```

Comment:
Also, if this times out, will it throw an exception? Or will the status value contain a timeout message for this key?